projects/core/src/occ/adapters/product/default-occ-product-config.ts

```diff
@@ -13,8 +13,6 @@ export const defaultOccProductConfig: OccConfig = {
           details:
             'products/${productCode}?fields=averageRating,stock(DEFAULT),description,availableForPickup,code,url,price(DEFAULT),numberOfReviews,manufacturer,categories(FULL),priceRange,multidimensional,tags,images(FULL)',
           attributes: 'products/${productCode}?fields=classifications',
-          variants:
```

Comment:
Could you mention removal of this scope from this code and the replacement using occProducts in the new default config for variants